gemrb/core/GameScript/GSUtils.cpp

```diff
@@ -1379,9 +1378,6 @@ void MoveBetweenAreasCore(Actor* actor, const ResRef &area, const Point &positio
 	if (actor->InParty) {
 		GameControl *gc=core->GetGameControl();
 		gc->SetScreenFlags(ScreenFlags::CenterOnActor, BitOp::OR);
-		if (newSong) {
-			game->ChangeSong(false, true);
-		}
```

Comment:
What's the alternative solution then? ChangeSong could easily be made to take an optional Map* parameter.

Reply:
No alternative solution needed. A GetMap which actually changes the area executes a ChangeSong afterwards.